ext/ffi/00_ffi.js

```diff
@@ -230,6 +230,28 @@
       this.#rid = core.opSync("op_ffi_load", { path, symbols });
 
       for (const symbol in symbols) {
+        if ("type" in symbols[symbol]) {
+          const type = symbols[symbol].type;
+          const name = symbols[symbol].name || symbol;
+          let value = core.opSync(
+            "op_ffi_get_static",
+            {
+              rid: this.#rid,
+              name,
+              type,
+            },
+          );
+          if (type === "pointer" || type === "u64") {
+            value = unpackU64(value);
+            if (type === "pointer") {
+              value = new UnsafePointer(value);
+            }
+          } else if (type === "i64") {
+            value = unpackI64(value);
+          }
+          this.symbols[symbol] = value;
```

Comment:
```suggestion
          Object.defineProperty(this.symbols, symbol, { get: () => value });
```

Reply:
Good point. Function fields should be a getter too, feel free to make that change in this PR. Also use the primordial for `Object.defineProperty` instead